apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/libs/arrangeSteps/arrangeSteps.ts

```diff
@@ -70,11 +70,11 @@ export function arrangeSteps(steps: TreeStepBlock[]): PositionMap {
   }
 
   blocks.forEach((row, index) => {
```

Comment:
can you change row here to column.
```suggestion
  blocks.forEach((column, index) => {
```